README.md

```diff
@@ -80,7 +80,7 @@ import re
 import sys
 
 
-def get_api_client(api_key_id, api_secret_file = None, private_key_string=None, endpoint="https://intersight.com"):
+def get_api_client(api_key_id, api_secret_file = None, private_key_string = None, proxy = None, endpoint="https://intersight.com"):
```

Comment:
Can you also add a section in the Readme under authentication segment, which shows how to set this proxy by making a call to this new `get_api_client`